pulp/Containerfile

```diff
@@ -6,7 +6,7 @@ ARG PULP_ANSIBLE_VERSION=""
 ARG PULP_CERTGUARD_VERSION=""
 ARG PULP_CONTAINER_VERSION=""
 ARG PULP_DEB_VERSION=""
-ARG PULP_FILE_VERSION=""
+ARG PULP_FILE_VERSION="<1.10"
```

Comment:
Hmm. I don't think, this is the soluttion here. I think we should make 1.10 unavailable on pypi (it has broken deps).
And if we want to fix anything here, make it `PULP_FILE_VERSION="!1.10"`.